### PR TITLE
Release Semantic Kernel for Python

### PR DESCRIPTION
**Pull request by dluc**
Merging Python SK into main branch + fixes/improvements

* Update READMEs
* Update notebooks to use pip package
* Add python getting started notebook
* Fix notebooks usage of Azure text completion
* Fix bug with skills definition
* New release version 0.2.2
* Fix .NET getting started notebook
* Rename notebook files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,31 +41,55 @@ peek at how the SDK is being built. SK has been released as open-source so that
...
+<a href="dotnet/README.md">Using Semantic Kernel in C#</a>.
+</div>
+
+<div style="display:flex;height:30px;padding:5px 0 5px 10px;">
+<img src="https://user-images.githubusercontent.com/371009/230673733-7a447d30-b48e-46e1-bd84-2b321c90649e.png" style="margin-right:12px" height="30"/>
+<a href="python/README.md">Using Semantic Kernel in Python</a>.
+</div>
+<br/>
+
+The quickest way to get started with the basics is to get an API key
```
[the remark this answers is not in the document]
Comment (alexchaomander) [reply README.md:59]: So suggestion would be to drop this part

**Review by awharrison-28 — CHANGES_REQUESTED** [review]
Added suggestions to fix tests for windows environments

File: python/DEV_SETUP.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+[Azure Open AI service key](https://learn.microsoft.com/azure/cognitive-services/openai/quickstart?pivots=rest-api)
+
+Copy those keys into a `.env` file (see the `.env.example` file):
+
+```
+OPENAI_API_KEY=""
+OPENAI_ORG_ID=""
+AZURE_OPENAI_API_KEY=""
+AZURE_OPENAI_ENDPOINT=""
+```
```
Comment (alexchaomander) [inline python/DEV_SETUP.md:18]: Do we need to add `AZURE_OPENAI_DEPLOYMENT_NAME` to match the sample .env file?

File: README.md [hunk 3]
```diff
@@ -80,59 +106,12 @@ working with Memory, and see how the kernel works.
...
-
-Console.WriteLine(output);
-
-// Output => Entropy is a funny thing,
-//           It's always increasing, no matter what we bring!
-```
+For a more hands-on overview, you can also check out the Jupyter C# and Python notebooks, starting
+from here:
+* [Getting Started with C# notebook](samples/notebooks/dotnet/Getting-Started-Notebook.ipynb)
+* [Getting Started with Python notebook](samples/notebooks/python/00-getting-started-notebook.ipynb)
```
Comment (alexchaomander) [inline README.md:114]: Invalid link. Instead: `samples/notebooks/python/00-getting-started.ipynb`

File: samples/notebooks/python/01-basic-loading-the-kernel.ipynb [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import semantic_kernel as sk\n",
+    "import semantic_kernel.ai.open_ai as sk_oai"
```
Comment (alexchaomander) [inline samples/notebooks/python/01-basic-loading-the-kernel.ipynb:37]: I would keep this consistent with the other notebooks and use `from semantic_kernel.ai.open_ai import AzureTextCompletion, OpenAITextCompletion`

File: README.md [hunk 5]
```diff
@@ -39,100 +41,79 @@ peek at how the SDK is being built. SK has been released as open-source so that
...
-
-## Get Started with Semantic Kernel ⚡
-
-Here is a quick example of how to use Semantic Kernel from a C# console app.
+  3. Semantic Kernel supports .NET Standard 2.0 and it's recommended using .NET 6+. However, some of
+     the examples in the repository require [.NET 7](https://dotnet.microsoft.com/download).
 
-1.  Create a new project, targeting .NET 6 or newer, and add the
-    `Microsoft.SemanticKernel` nuget package:
+## Notebooks ⚡
```
Comment (alexchaomander) [inline README.md:107]: Rename to Jupyter Notebooks 📚

File: README.md [hunk 6]
```diff
@@ -39,100 +41,79 @@ peek at how the SDK is being built. SK has been released as open-source so that
...
 
-        dotnet add package Microsoft.SemanticKernel --prerelease
+For a more hands-on overview, you can also check out the Jupyter C# and Python notebooks, starting
```
Comment (alexchaomander) [inline README.md:109]: "check out the C# and Python Jupyter Notebooks"

File: samples/notebooks/python/04-context-variables-chat.ipynb [hunk 7]
```diff
@@ -0,0 +1,246 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "92f69b34",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!python -m pip install ../../../python"
```
Comment (alexchaomander) [inline samples/notebooks/python/04-context-variables-chat.ipynb:28]: Change to !python -m pip install -r requirements.txt

**Review by awharrison-28 — DISMISSED** [review]

**Pull request merged**